### PR TITLE
Uncapitalize words as per new guidelines

### PR DESCRIPTION
**Pull request by lmoureaux**

### CODE REVIEW

File: docs/Coding/hacking.rst [hunk 1]
```diff
@@ -591,11 +591,11 @@ With an isometric view, this looks like:
     |                            (nat_x, nat_y+2)
 
 
-Neither is particularly good for a global Map operation such as Map wrapping or conversions to or from Map
-indexes. Something better is needed.
+Neither is particularly good for a global map operation such as map wrapping or conversions to or from map
+indexes. something better is needed.
```
Comment (daavko) [inline docs/Coding/hacking.rst:595]: Typo: "something" should be left capitalized, it begins a sentence

File: docs/General/Manuals/client-manual.rst [hunk 2]
```diff
@@ -281,19 +281,19 @@ Start Scenario Game
...
 When you click on the :guilabel:`Start Scenario Game` button, a dialog box will appear as in
-:numref:`Scenarios Dialog` below. You can click on a scenario to select it and see information about the
+:numref:`Scenarios dialog` below. You can click on a scenario to select it and see information about the
 selected scenario on the panel to the right side. When you have found the scenario you want to run, click on
 the :guilabel:`Load Scenario` button. This will bring up the new game dialog as shown in :numref:`Start New
-Game Dialog` above. When ready, click :guilabel:`Start` to begin the scenario game.
+Game dialog` above. When ready, click :guilabel:`Start` to begin the scenario game.
 
-.. _Scenarios Dialog:
+.. .. _Scenarios dialog:
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:289]: Why the extra two dots here?

File: docs/General/Manuals/client-manual.rst [hunk 3]
```diff
@@ -1045,21 +1045,21 @@ Map View
...
 
-.. _Map View Button:
+.. _Map view button:
 .. figure:: ../../_static/images/gui-elements/top-bar-map.png
   :align: center
   :alt: Freeciv21 Map
   :figclass: align-center
 
-  Map View Button
+  Map view button
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1057]: Why is "View" uncapitalized here? Everywhere else it's called "Map View"

File: docs/General/Manuals/client-manual.rst [hunk 4]
```diff
@@ -1045,21 +1045,21 @@ Map View
...
 
 
 If you hover your mouse over the :guilabel:`Map View` button, a pop up widget will appear and give you
 information about your nation. The pop up widget shows: Nationality, Total Population, Year (Turn Number),
-Gold (Surplus/Deficit), and Tax Rates.
+gold (Surplus/Deficit), and Tax Rates.
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1062]: I'd leave this capitalized, it's capitalized in the pop-up as well

File: docs/General/Manuals/client-manual.rst [hunk 5]
```diff
@@ -1083,15 +1083,15 @@ Units View
...
 
-.. _Units View Button:
+.. _Units view button:
 .. figure:: ../../_static/images/gui-elements/top-bar-units.png
   :align: center
   :alt: Freeciv21 Units
   :figclass: align-center
 
-  Units View Button
+  Units view button
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1094]: Same as before, this is called "Units View" everywhere, I don't see why "View" is uncapitalized

File: docs/General/Manuals/client-manual.rst [hunk 6]
```diff
@@ -1123,15 +1123,15 @@ Cities View
...
 
-.. _Cities View Button:
+.. _Cities view button:
 .. figure:: ../../_static/images/gui-elements/top-bar-cities.png
   :align: center
   :alt: Freeciv21 Cities
   :figclass: align-center
 
-  Cities View Button
+  Cities view button
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1134]: Also here, "Cities View"

File: docs/General/Manuals/client-manual.rst [hunk 7]
```diff
@@ -1160,20 +1160,20 @@ Nations and Diplomacy View
...
   :scale: 50%
   :alt: Freeciv21 Nations
   :figclass: align-center
 
-  Nations and Diplomacy View Button
+  Nations and Diplomacy view button
 
 
-:numref:`Nations and Diplomacy View Button with Diplomacy View Open` gives a sample of the
+:numref:`Nations and Diplomacy view button with Diplomacy View open` gives a sample of the
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1176]: Here as well, it's called "Nations and Diplomacy View", so "View" should be capitalized

File: docs/General/Manuals/client-manual.rst [hunk 8]
```diff
@@ -1388,21 +1388,21 @@ you what was sold where.
 Tax Rates View
 ^^^^^^^^^^^^^^
 
-The :guilabel:`Tax Rates View` on the `Top Function Bar`_ shows what percentage of Gold, Science, and Luxury
-Goods your nation is set at. :numref:`Tax Rates View Button` shows an example of the
+The :guilabel:`Tax Rates View` on the `Top Function Bar`_ shows what percentage of gold, Science, and luxury
+goods your nation is set at. :numref:`Tax Rates view button` shows an example of the
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1392]: This capitalization is inconsistent, "science" should be lowercase as well

File: docs/General/Manuals/client-manual.rst [hunk 9]
```diff
@@ -1691,7 +1692,7 @@ and Wonders.
 
 Overall happiness of the citizens in your cities depends heavily on all of these factors. Certain forms of
 government have varying degrees of empire size penalties and as your empire grows you will have to deal with
-the negative consequences of managing a large empire and the unhappiness it creates. The amount of Luxury
+the negative consequences of managing a large empire and the unhappiness it creates. The amount of luxury
 Goods you are producing as part of your `Tax Rates View`_ will aid this problem. Certain city improvements can
```
Comment (daavko) [inline docs/General/Manuals/client-manual.rst:1696]: This is now "luxury Goods", should it be all lowercase?

**Review by daavko — COMMENTED** [review]
I stopped mentioning individual occurences because it got a little much:
In `docs/General/Manuals/client-manual.rst`, some "Xyz View" are now uncapitalized to "Xyz view", even though they're still called "Xyz View" in other places. This should be consistent, I'd vote for leaving it as capital "View". But I don't mind either way, the main point is consistency.

File: docs/Coding/hacking.rst (hunk 1 above)
Comment (jwrober) [reply docs/Coding/hacking.rst:595]: Distinct GUI elements are supposed to be capitalized. Map View, Cities View, etc. Messages Widget, Chat Widget, etc. Production Tab, General Tab, etc. should all be capitalized.

File: docs/General/Manuals/client-manual.rst (hunk 9 above)
Comment (jwrober) [reply docs/General/Manuals/client-manual.rst:1696]: Gold, Tax(es) and Luxury Goods are specific game elements and also very prominent in the GUI and I think should be capitalized.

File: docs/Coding/hacking.rst (hunk 1 above)
Comment (lmoureaux) [reply docs/Coding/hacking.rst:595]: > Distinct GUI elements are supposed to be capitalized. Map View, Cities View, etc. Messages Widget, Chat Widget, etc. Production Tab, General Tab, etc. should all be capitalized.

Not per the guidelines

Comment (jwrober) [reply docs/Coding/hacking.rst:595]: Ah, it got excised in PR #1334 by accident. I didn't catch that during review.

**Comment by lmoureaux** [comment]
Following what's done in the Wesnoth manual, the names of GUI elements shouldn't be capitalized. However I kept them capitalized in figure captions under the (new) rule that capitalization should be used when a caption is essentially a title.

I didn't change back the capitalization in `:alt:` because I think they should be changed more deeply -- currently they merely duplicate information already available in the caption, and it looks bad in a text-only browser.

**Review by jwrober — APPROVED** [review]

**Pull request merged**